lib/ansible/modules/user.py

```diff
@@ -1169,6 +1184,8 @@ def ssh_key_gen(self):
             else:
                 return (None, 'Key already exists, use "force: yes" to overwrite', '')
         cmd = [self.module.get_bin_path('ssh-keygen', True)]
+        cmd.append('-a')
+        cmd.append(self.ssh_key_trails)
```

Comment:
his is still a trail ...

Reply:
Sorry for that. :-(. Fixed it.

Reply:
no need to be sorry (i'm the tpyoe king!)